src/Symfony/Component/Form/Extension/Core/ChoiceList/LazyChoiceList.php

```diff
@@ -31,6 +35,11 @@
      */
     private $choiceList;
 
+    public function __construct()
+    {
+        trigger_error('The '.__CLASS__.' class is deprecated since version 2.7 and will be removed in 3.0. Use Symfony\Component\Form\ChoiceList\LazyChoiceList instead.', E_USER_DEPRECATED);
```

Comment:
this should be moved near the namespace declaration so that it is triggered even when some people overwrite the constructor in their child class